test/Negative.test.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
if you run the (new) linter, you should catch anything that shouldn't be in this diff